web/webkit/src/main/scala/net/liftweb/http/LiftSession.scala

```diff
@@ -2372,7 +2395,20 @@ class LiftSession(private[http] val _contextPath: String, val underlyingId: Stri
     }
   }
 
-  private def buildAndStoreComet[T <: LiftCometActor](newCometFn: (CometCreationInfo)=>Box[T])(creationInfo: CometCreationInfo): Box[T] = {
+  /**
+   * Need to add your own CometActor to a Session/Page? You might 'cause creating
+   * a CometActor programmatically within a snippet may be between that referring
+   * to a CometActor by class name.
+   *
+   * This is dangerous stuff, so use with caution.
+   *
+   * @param newCometFn a Function that takes `CometCreateInfo` and returns a CometActor
+   *                   instance
+   * @param creationInfo the info to send to the above function
+   * @tparam T A LiftCometActor or better
+   * @return The created, registered, etc. CometActor
+   */
+  def buildAndStoreComet[T <: LiftCometActor](newCometFn: (CometCreationInfo)=>Box[T])(creationInfo: CometCreationInfo): Box[T] = {
```

Comment:
I needed a place to dynamically create CometActors rather than refer to a CometActor that had been compiled with the project. Now how can one dynamically subclass a JVM class without compiling it with the project? Yep... it was some of the Scala/Clojure/Lift/Reagent inter-op stuff I've been doing.

But more broadly, I realized that there might be times even in pure Scala-land that one might want to do a:

```
new CometActor { ..... }
```

Rather than creating a separate CometActor subclass in a file with a name.